UE4SS/src/LuaType/LuaUObject.cpp

```diff
@@ -684,6 +690,16 @@ namespace RC::LuaType
                         field_type_fname.ToString()
                         );
             }
+        };
+
+        LuaCustomProperty::StaticStorage::property_list.for_each(script_struct, [&](LuaCustomProperty const& lua_property) {
+            handle_property((Unreal::FProperty*)lua_property.m_property.get(), to_utf8_string(lua_property.m_name));
```

Comment:
```suggestion
            handle_property(lua_property.m_property.get(), to_utf8_string(lua_property.m_name));
```
We don't allow C-casts, and you shouldn't need to cast from LuaCustomProperty* to FProperty* at all.